.gitignore

```diff
@@ -167,6 +167,8 @@ ide/index_urls.txt
 
 /user-contrib/*
 !/user-contrib/Ltac2
+/user-contrib/Ltac2/dune
```

Comment:
Maybe put it at the end, near `plugins/*/dune`?

Reply:
Makes sense. Done.